test/fake.js

```diff
@@ -79,6 +79,8 @@ module.exports.newUserDataHex = function() {
     createdAt: Date.now()
   }
 
+  data.unlockCode = hex16()
```

Comment:
Can you add an equivalent line to the `newUserDataBuffer()` function too? This is only used in the prune_tokens test but as a matter of completeness.

Note: I'm totally to blame here for the duplicate functions and I'll look into having just one function generate the data and convert it into a Buffer/Hex equivalent. Sorry.